test/razorpay/test_customer.rb

```diff
@@ -65,5 +67,56 @@ def test_customer_delete_token
       token = Razorpay::Customer.fetch(@customer_id).deleteToken("token_FHfn3rIiM1Z8nr")
       assert token.deleted
     end
+
+    def test_customer_add_bank_account
```

Comment:
please add test cases for error scenarios as well for each of the APIs added.

Reply:
added error test cases